api/modules/currency/constants.py

```diff
@@ -3,3 +3,4 @@
 Reference Docs: https://free.currencyconverterapi.com/
 """
 CURRENCY_CONVERTER_API_URL = 'https://free.currencyconverterapi.com/api/v5/convert?q={}'
+CURRENCY_VALUE_DATE_API_URL = 'http://currencies.apps.grandtrunk.net/getrate/{0}/{1}/{2}'
```

Comment:
This should be expecting 4 parameters, right?
`start_date`, `end_date`, `source_currency_code`, `target_currency_code`

Reply:
Thing is if I use the other URL then it skips through dates which you didn't want. That's why I used a loop

Reply:
If the API response is something like this
```
10-07-2018 1.4
12-07-2018 1.5
```
So for 10-07-2018 to 14-07-2018 you can have something like 
```
[
    1.4,
    1.4,
    1.5,
    1.5
    1.5
]
```

Reply:
But the API request to the server is taking 4 parameters. From the server to the currency API is using 3.